contracts/airdrops/Existing1155NftDrop.sol

```diff
@@ -4,61 +4,46 @@ pragma solidity ^0.8.14;
 import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
 import "@openzeppelin/contracts/token/ERC1155/IERC1155.sol";
 import "@openzeppelin/contracts/token/ERC1155/IERC1155Receiver.sol";
-import "@openzeppelin/contracts/access/Ownable.sol";
 import "../interfaces/AirdropMerkleProof.sol";
-import "../interfaces/AirdropInfo.sol";
 import "../interfaces/IAirBroFactory.sol";
 
 /// @title Airdrops existing ERC1155 tokens for airdrop recipients
-contract Existing1155NftDrop is AirdropInfo, AirdropMerkleProof, IERC1155Receiver, Ownable {
+contract Existing1155NftDrop is AirdropMerkleProof, IERC1155Receiver {
```

Comment:
Since in all airdrop contract share a lot of data in constructor you can make an abstract contract with a constructor like this, and then have contrats extend it
```
        rewardedNft = IERC721(_rewardedNft);
        tokensPerClaim = _tokensPerClaim;
        totalAirdropAmount = _totalAirdropAmount;
        airdropDuration = _airdropDuration * 1 days;
        airdropStartTime = block.timestamp;
        airdropFinishTime = block.timestamp + airdropDuration;
```
(pls double what data is shared)

Reply:
No need for `AirdropMerkleProof`? you aren't checking proof anywhere

Reply:
Created two abstract parent contracts [(see commit here)](https://github.com/MVPWorkshop/Airbro-contracts/commit/9d696e0432bb4ea81131bca735bed7332aad778f) for shared state constructor initialisation. 

Looking for more room to create more shared contracts like this.

Also removed AirdropMerkleProof from all contracts that do not implement it.